style.css

```diff
@@ -0,0 +1,653 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'lato', sans-serif;
```

Comment:
Nice font, happy to see that you're using `Lato`, one of the fonts of the original design 💯 but make sure to also use `COCOGOOSE` when needed in the project. 🤓  👇🏽 

![Screenshot 2023-04-18 at 22 33 58](https://user-images.githubusercontent.com/45739963/232951825-0c293d50-a098-4a6c-9da8-ed4ddf51444d.png)

Reply:
@didierganthier Thank you for your feedback, but Cocogoose font is not in the requirements, yes Lato font is, but not the other font

Reply:
@StevenWafeek Give me one sec to double check

Reply:
You're right @StevenWafeek, I mentioned it as it was part of the design guidelines but I'm marking it as optional, thanks for pointing that out. 👍🏽